doc/manual/types.rst

```diff
@@ -432,7 +434,7 @@ You can also change the values as one would expect:
 .. doctest::
 
     julia> foo.qux = 2
-    2.0
+    2
```

Comment:
I believe this is a bug on 0.4 https://github.com/JuliaLang/julia/issues/14691 . It's a breaking change so I'm not sure if it should be backported.